pset-iterators.js

```diff
@@ -43,6 +43,14 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find(function (president) {
+  return president.party === 'Whig'
+})
+
+console.log(firstWhigPresident.president)
+
```

Comment:
@thetripler Good job here using the find method to get back the first president of the whig party.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)